dace/frontend/python/memlet_parser.py

```diff
@@ -136,6 +136,8 @@ def parse_memlet_subset(array: data.Data, node: Union[ast.Name, ast.Subscript],
     else:  # Use entire range
         subset = _ndslice_to_subset(ndslice)
 
+    if isinstance(subset, subsets.Indices):
+        subset = subsets.Range([(i, i, 1) for i in subset])
```

Comment:
we have `subsets.Range.from_indices` now. Separate PR though.

Reply:
Cool, we can update it when we merge.